src/main/features/response/routes/statement-of-means/debts.ts

```diff
@@ -0,0 +1,58 @@
+import * as express from 'express'
+
+import { Paths, StatementOfMeansPaths } from 'response/paths'
+import { Form } from 'forms/form'
+import { FormValidator } from 'app/forms/validation/formValidator'
+import { ErrorHandling } from 'common/errorHandling'
+import { DraftService } from 'services/draftService'
+import { User } from 'idam/user'
+import { RoutablePath } from 'common/router/routablePath'
+import { FeatureToggleGuard } from 'guards/featureToggleGuard'
+import { Debts } from 'response/form/models/statement-of-means/debts'
+
+const page: RoutablePath = StatementOfMeansPaths.debtsPage
+
+function renderView (form: Form<Debts>, res: express.Response): void {
+  res.render(page.associatedView, { form: form })
+}
+
+function actionHandler (req: express.Request, res: express.Response, next: express.NextFunction): void {
```

Comment:
looks like a generic actionHandler that could be extracted? up to you

Reply:
I will

Reply:
not much gain, a lot of pain ;)